php/Terminus/Models/Site.php

```diff
@@ -483,7 +483,9 @@ public function info($key = null) {
       }
     }
     if (!is_null($info['created'])) {
-      $info['created'] = date(TERMINUS_DATE_FORMAT, $info['created']);
+      if (is_numeric($info['created'])) {
```

Comment:
It would be more efficient to combine these statements with
` if (!is_null($info['created']) && is_numeric($info['created'])) {`